railties/lib/rails/commands/secrets/secrets_command.rb

```diff
@@ -0,0 +1,56 @@
+require "active_support"
+require "active_support/core_ext/string/strip"
+require "rails/generators/rails/app/app_generator"
+
+require "rails/secrets"
+
+module Rails
+  module Command
+    class SecretsCommand < Rails::Command::Base # :nodoc:
+      def help
+        super
+        puts self.class.desc
+      end
+
+      def perform
+        require_application_and_environment!
+
+        setup
+        edit
+      end
+
+      def setup
+        require_application_and_environment!
+
+        Rails::Generators::AppGenerator.new([ Rails.root ]).setup_encrypted_secrets
+      end
+
+      def edit
```

Comment:
Neatness: this change also spills into improvements to the command infrastructure.

Previously we couldn't handle `bin/rails secrets`, `bin/rails secrets:setup` and `bin/rails secrets:edit` in one command file. But with some "smarter" lookup we can 💪

(I say smarter because the command lookups could use some simplifying but that's too much for this PR. So pour more coal into this 🚂, less changing lanes.)